### PR TITLE
Be more consistent when referring to acd namespace

### PR DESCRIPTION
**Pull request by cagrev**

### CODE REVIEW

**Review by herbeck558 — APPROVED** [review]
LGTM

File: src/pages/installing/installing.md [hunk 1]
```diff
@@ -137,7 +137,7 @@ To add the pull secret to individual ACD operand service accounts:
        --docker-username=<username> \
        --docker-password=<password> \
        --docker-email=<email_address> \
-       --namespace=<namespace>
+       --namespace=${acd_namespace}
```
Comment (dlangst) [inline src/pages/installing/installing.md:140]: Do we want to add a note about setting the ${acd_namespace} variable first?  The 'Set Up Namespace' page has some details, but I'm thinking of the case where the reader might not read each page sequentially.

Comment (cagrev) [reply src/pages/installing/installing.md:140]: I thought about that, but didn't think it was necessary on every page where it's used, especially for the installing piece. They'd be creating the namespace for a new install anyway so quite likely already visited the set up namespace page. We do also have some similar guidance on the migration pages so I felt we were covered. If you think it's necessary, go ahead and add it where you think it's needed for clarification.

File: src/pages/installing/verifying.md [hunk 2]
```diff
@@ -11,32 +11,32 @@ toc: true
 List the pods deployed and look for the operator pod showing it is ready with a status of running.
 
 ```
-oc get pods --namespace <namespace> | grep controller
+oc get pods --namespace ${acd_namespace} | grep controller
```
Comment (dlangst) [inline src/pages/installing/verifying.md:14]: Same comment here.  Perhaps a quick note about where/when ${acd_namespace} variable gets set if someone doesn't read each page sequentially.

**Review by dlangst — COMMENTED** [review]
Looks good.  Added a comment.  Your call if we want the extra detail or not.

**Pull request merged**